src/windowmanagement/WindowLayout/WindowLayout.tsx

```diff
@@ -69,7 +69,7 @@ export function WindowLayout() {
 
   // Populate default visible items for taskbar
   useEffect(() => {
-    const defaultVisibleMenuItems = menuItemsDB
+    const defaultVisibleMenuItems = menuItemsData
       .map((menuItem) => {
         if (menuItem.defaultVisible) {
           return menuItem.componentID;
```

Comment:
No need to use map here, you can just filter directly

Reply:
lol 100% smarter than whatever this was